skyvern/webeye/scraper/domUtils.js

```diff
@@ -223,6 +223,11 @@ function isElementVisible(element) {
     return false;
   }
 
+  if (element.hasAttribute("aria-hidden")) {
+    const ariaHidden = element.getAttribute("aria-hidden").toLowerCase();
+    return ariaHidden === "false";
```

Comment:
except aria-hiden="False" or aria-hiden="false", all aria-hidden values should be considered as hidden, like `aria-hidden`, `aria-hidden=""`, `aria-hidden="true"`, `aria-hidden="True"`